### ISSUE TITLE
Mutate API integration

### ISSUE THREAD

**Issue by CaryLandholt** [jbielick/faktory_worker_node#36]
Will you be integrating the Mutate API?

https://github.com/contribsys/faktory/wiki/Mutate-API
https://github.com/contribsys/faktory/blob/master/client/mutate.go

**Comment by jbielick** [comment]
Hey Cary!

I’m open to it—mostly waiting for a request for it. We’re you planning on using the mutate API from the node client?

**Comment by CaryLandholt** [comment]
I am, yes. Was going to play around with your “send” command first. Fhanks

**Comment by jbielick** [comment]
I'll start some work to add it.

If it wasn't clear / documented, you indeed should be able to use `send` in the mean time like so:

```js
const instruction = { cmd: "discard", target: "scheduled", filter: { jobtype: "QuickbooksSyncJob" } };
await client.send(['MUTATE', Client.encode(instruction)])
```

**Comment by CaryLandholt** [comment]
Excellent!

**Comment by jbielick** [comment]
The Mutate API is implemented in [`v3.1.0`](https://www.npmjs.com/package/faktory-worker/v/3.1.0)

There are [some docs here](https://github.com/jbielick/faktory_worker_node/blob/master/docs/api.md#Mutation) for the mutation API. 

I'll write some more at some point. Let me know if that works for you.

```js
const client = faktory.connect();

await client.retries.ofType('JobType').requeue();
await client.scheduled.withJids('123').discard();
await client.dead.clear();
```

etc...

**Comment by CaryLandholt** [comment]
Thanks!
